staging/src/k8s.io/apiserver/pkg/util/flowcontrol/reqmgmt.go

```diff
@@ -251,20 +252,19 @@ func (reqMgmt *requestManagementSystem) Wait(requestDigest RequestDigest) (bool,
 		// 3. executing
 		ps := rmState.priorityLevelStates[matchingpriorityLevelName]
 		if ps.config.Exempt {
-			klog.V(7).Infof("Serving requestInfo=%v, user=%v, fs=%q without delay", requestDigest.RequestInfo, requestDigest.User, fs.Name)
+			klog.V(7).Infof("Serving %v without delay", requestDigest)
 			return true, func() {}
 		}
 		quiescent, execute, afterExecute := ps.queues.Wait(hashValue, ps.config.HandSize)
 		if quiescent {
-			klog.V(5).Infof("Request requestInfo=%v, user=%v, fs=%q landed in timing splinter, re-classifying", requestDigest.RequestInfo, requestDigest.User, fs.Name)
+			klog.V(5).Infof("Request %v landed in timing splinter, re-classifying", requestDigest)
 			continue
 		}
-		if execute {
-			klog.V(7).Infof("Serving requestInfo=%v, user=%v, fs=%q after fair queuing", requestDigest.RequestInfo, requestDigest.User, fs.Name)
-		} else {
-			klog.V(7).Infof("Rejecting requestInfo=%v, user=%v, fs=%q after fair queuing", requestDigest.RequestInfo, requestDigest.User, fs.Name)
+		startExecutionTime := time.Now()
+		return execute, func() {
+			afterExecute()
```

Comment:
I would put this after the observation of the execution duration, the `afterExecute()` is about other work not the request just finished.

Reply:
done

Reply:
This has been fixed and can be marked resolved.